test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -58,6 +70,20 @@
    :web-router-service {:puppetlabs.trapperkeeper.services.metrics.metrics-service/metrics-webservice
                         "/metrics"}})
 
+(def auth-config
+  {:authorization
+   {:version 1
```

Comment:
did you just pick `1` arbitrarily?

Reply:
https://puppet.com/docs/puppetserver/latest/config_file_auth.html#version